app/views/users/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<%# #  <form accept-charset="UTF-8" action="/users" method="post">
+#     <label for="username">Username:</label>
+#     <input id="username" name="user[name]" type="text" />
+      <label for="email">Email:</label>
+#     <input id="email" name="user[email]" type="email" />
+      <label for="password">Password:</label>
+#     <input id="password" name="user[password]" type="password" />
+#     <input name="submit" type="submit" value="Submit" />
+#     <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+   <%# </form> %> %>
+
+<%# <%= form_tag("/users_path", method: "post") do %> %>
```

Comment:
Please remove the `/` it's `users_path` not `/users_path` it's a helper method that will help you navigate to the root path easily.